torchvision/prototype/transforms/_color.py

```diff
@@ -84,30 +84,6 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         return output
 
 
-class _RandomChannelShuffle(Transform):
```

Comment:
Since this PR removes the last remnants of the former idiom to use primitive transforms in other transforms, I've moved this functionality into a method of `RandomPhotometricDistort`.